src/component/contents/DraftEditorContents.react.js

```diff
@@ -197,21 +199,80 @@ class DraftEditorContents extends React.Component {
         wrapperTemplate,
         key,
         offsetKey,
+        draftWrapperTemplate: configForType.wrapper,
+        customWrapperTemplate: customWrapperTemplate,
       });
 
-      if (wrapperTemplate) {
+      if (draftWrapperTemplate || customWrapperTemplate) {
         currentDepth = block.getDepth();
       } else {
         currentDepth = null;
       }
       lastWrapperTemplate = wrapperTemplate;
+      lastCustomWrapperTemplate = customWrapperTemplate;
     }
 
-    // Group contiguous runs of blocks that have the same wrapperTemplate
+    // Group contigious runs of blocks
```

Comment:
Nit: I think the `contiguous` spelling is the one we want!

Reply:
Oops, haven't added my spell check plugin to vscode yet! Thanks!!